scripts/ubuntu-bartender/ubuntu-bartender

```diff
@@ -107,9 +107,23 @@ AWS_ARM_BUILD=false
 
 # MULTIPASS SPECIFIC DEFAULTS
 MULTIPASS_DISK_SIZE=50G
-MULTIPASS_MEM_SIZE=8G
+# MULTIPASS_MEM_SIZE=8G  # you can optionally override this
```

Comment:
This can be removed now with line 125

Reply:
I've updated the comment to remove the "commented out" part, but I don't want to *remove* the comment altogether. This was where I found the solution to my problem to begin with; I'd rather not remove information.